deploy/operator.yaml

```diff
@@ -4,44 +4,41 @@ metadata:
   name: kie-cloud-operator
 spec:
   replicas: 1
+  selector:
```

Comment:
Why the (non-functional) changes to this (and the role) file?

Reply:
This file is now generated as well and, with this commit, it's just a reordering of some the fields.